transport-native-io_uring/src/main/java/io/netty/channel/uring/AbstractIOUringChannel.java

```diff
@@ -163,6 +163,11 @@ public FileDescriptor fd() {
     @Override
     protected abstract AbstractUringUnsafe newUnsafe();
 
+    @Override
+    public AbstractUringUnsafe unsafe() {
```

Comment:
Hmm it’s kind of odd to expose a package-private class in a public method imho

Reply:
@normanmaurer sure I can revert this if you prefer. I don't think it's really exposed though since the `AbstractIOUringChannel` class itself is also package-private. So anyone calling from outside would be operating on the superclass version of it anyhow which returns `Unsafe` and so would not see or be aware of this... wdyt?

Reply:
Yeah let’s do that

Reply:
done